protocol/x/prices/keeper/market.go

```diff
@@ -48,6 +48,15 @@ func (k Keeper) CreateMarket(
 			)
 		}
 	}
+	// check that the market exists in market map
```

Comment:
nit: Isn't this validation only checking if the market pair string is formatted as a valid currency pair? There's no query into market map here.